ansible/Makefile

```diff
@@ -0,0 +1,7 @@
+REPO?=kubevirt
+PROJECT?=kubevirt-ansible
+VERSION?=master
+ROLE?=cdi
+
+fetch:
+	wget -qO- https://api.github.com/repos/$(REPO)/$(PROJECT)/tarball/$(VERSION) | tar xzv --wildcards "*/roles/$(ROLE)" --strip-components=1
```

Comment:
probably want to add this file to .gitignore

Reply:
The archive is extracted on the fly but it's worth to add the whole ansible/roles/

Reply:
right, that's what I meant.  We aren't tracking changes so it should be ignored.